### DISCUSSION TITLE
Plan to update Windows version to 21325 or above

### DISCUSSION THREAD

**Discussion by SunandaChowdhary** [actions/virtual-environments#10508] [General questions]
20348 is the latest version for Windows Server 2022. Is there any plan to upgrade windows version? We need to run automation for WAM scenario which was introduced in version 21325 onwards.

**Comment by erik-bershel** [comment]
Hey @SunandaChowdhary!

We use Azure Windows Server 2022 image:
```
  image_offer                            = "WindowsServer"
  image_publisher                        = "MicrosoftWindowsServer"
  image_sku                              = "2022-Datacenter"
```

You can expect updates to appear here about a week after the base image is updated.

**Reply by SunandaChowdhary** [reply]
Hello @erik-bershel , I know that Windows 2022 image used currently has windows version 20348. My question is when will windows version be upgraded to say 21325 and above? Where can I see the schedule for version update in images? Where can I request for an upgraded image?

**Reply by erik-bershel** [reply]
> My question is when will windows version be upgraded to say 21325 and above?

Cannot be predicted. In about a week after the base image in the Azure Gallery is updated by the corresponding team.

> Where can I see the schedule for version update in images?

[We typically deploy weekly updates to the software on the runner images.](https://github.com/actions/runner-images?tab=readme-ov-file#image-releases)

> Where can I request for an upgraded image?

We won't accept this request since we have no control over the Azure Gallery images.

**Reply by erik-bershel** [reply]
In case you are asking about Windows Server 2025 release - we cannot share such a plans publicly. We will announce it when `beta` hosted image available.
